force-app/lwc/signals/core.js

```diff
@@ -28,8 +32,10 @@ const READY = Symbol("READY");
  * ```
  *
  * @param fn The function to execute
+ * @param props Options to configure the effect
  */
-function $effect(fn) {
+function $effect(fn, props) {
```

Comment:
why props and not options? like already being used for $signal

Reply:
Good call, I forgot I had called it that somewhere else

Reply:
I am fine with either but it should be the same for both